components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -4345,6 +4458,62 @@ components:
         ref:
           type: string
           example: '/t/wso2.com/api/server/v1/organizations/b4526d91-a8bf-43d2-8b14-c548cf73065b'
+    LoginFlowGenerateRequest:
+      type: object
+      properties:
+        available_authenticators:
```

Comment:
Be consistent with the existing naming convention. Fix other places in the PR as well.